src/components/CityWeatherShortcut/CityWeatherShortcut.tsx

```diff
@@ -31,6 +33,15 @@ const CityWeatherShortcut = ({ cityName, button }: Props) => {
     const { condition, temp_c } = data.current;
     const roundedTemperature = Math.round(temp_c);
     const { name } = data.location;
+    let button: ReactNode = <></>;
+
+    if (type === ButtonTypes.Heart) {
+      button = <FavoritesButton cityName={cityName} size={20} />;
+    }
+
+    if (type === ButtonTypes.Cross) {
+      //
+    }
```

Comment:
**suggestion** i would suggest to add `type` prop to `FavoriteButton` with default type `Heart` and later you can just pass `type` prop from `CityWeatherShortcut` to `FavoriteButton`

Reply:
But it will be inconsistent with mockups

Reply:
@bednarskimichal what do you think?

Reply:
I think that you could pass `type` to `FavoriteButton` and then inside it handle different types. You don't need to handle that in `CityWeatherShortcut`

Reply:
Okay

Reply:
But if button is not type of heart i won't use this icon, because it won't pass manual tests